### DISCUSSION TITLE
Support for max.request.size producer config

### DISCUSSION THREAD

**Discussion by gagangoku** [segmentio/kafka-go#1160] [Q&A]
I want to send large messages through Kafka (>10 MB). 

[tutorial 1](https://www.baeldung.com/java-kafka-send-large-message) and [tutorial 2](https://www.conduktor.io/kafka/how-to-send-large-messages-in-apache-kafka/) suggest that the 1st step is to set the `max.request.size` property in producer config. But i'm not finding the option in the Writer config.

Is this supported ?

**Comment by nko-sh** [comment]
I had the same issue today.
Apparently, the `BatchBytes` property in the writer configuration changes `max.request.size`.
